katas/joaquin-gioffre/main/Week-02/does-my-number-look-big-in-this/IsNarcissistic.cs

```diff
@@ -0,0 +1,18 @@
+namespace Week2;
+
+using System;
+
+public static class DoesMyNumberLookBigInThis
+{
+    public static bool IsNarcissistic(int value)
+    {
+        string number = value.ToString();
+        int length = number.Length;
+        int sum = 0;
+        foreach (char digit in number)
+        {
+            sum += (int) Math.Pow(digit - '0', length);
```

Comment:
What is digit - '0' for? is it a way to parse a char to an int?

Reply:
Yes, it's a way to get the numerical value of a character according to the ASCII table https://www.asciitable.com.

The character '0' has a numerical value of 48 in the table. By subtracting '0' from the "digit" variable, the numerical value of the digit is obtained, e.g. if the digit is '5', it will evaluate 53 - 48.

Reply:
Awesome, so better variable naming can help understanding this better, something like this:
- Instead of digit -> digitAsciiCode
- And again '0' could be stored on a constant called, ZERO_ASCII_CODE

And well, actually a more readable way of doing this is: char.GetNumericValue(digitChar)

Reply:
Cool, I just added the constant and updated the naming. Thanks!